### PR TITLE
CCM-2615: added NHS Number link to docs in spec

### PR DESCRIPTION
**Pull request by DavidFueryBJSS**
## Summary
Added NHSnumber link info to status 400 info in:
https://digital.nhs.uk/developer/api-catalogue/nhs-notify#post-/v1/message-batches
https://digital.nhs.uk/developer/api-catalogue/nhs-notify#post-/v1/messages

## Reviews Required
* [x] Dev
* [x] Test
* [ ] Tech Author
* [ ] Product Owner

## Checklist
* [x] Brief description of work completed, and any technical decisions made as part of the PR
* [x] PR link added as a comment to the relevant JIRA ticket
* [ ] Branch deployed to a dynamic env - link to deployment pipeline
* [x] PR link shared on Slack and/or Teams
* [x] 2 reviews received
* [x] Tester approval

### CODE REVIEW

File: specification/schemas/types/LinksError.yaml [hunk 1]
```diff
@@ -3,8 +3,7 @@ additionalProperties: false
 properties:
   about:
     type: string
-    enum:
-      - 'https://digital.nhs.uk/developer/api-catalogue/communications-manager'
     format: uri
     example: 'https://digital.nhs.uk/developer/api-catalogue/communications-manager'
```
Comment (tdroza-nhs) [inline specification/schemas/types/LinksError.yaml:7]: Although there's currently a redirect in place, can you update these to the new nhs-notify url please? e.g. `https://digital.nhs.uk/developer/api-catalogue/nhs-notify`

File: specification/schemas/types/LinksErrorNHSNumber.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+type: object
+additionalProperties: false
+properties:
+  about:
+    type: string
+    format: uri
+    example: 'https://digital.nhs.uk/developer/api-catalogue/communications-manager'
```
Comment (tdroza-nhs) [inline specification/schemas/types/LinksErrorNHSNumber.yaml:7]: As above re `communications-manager` -> `nhs-notify` urls

File: specification/schemas/types/LinksError.yaml (hunk 1 above)
Comment (tdroza-nhs) [reply specification/schemas/types/LinksError.yaml:7]: Actually, given that this is an example of a response message that we return, can you check whether we've updated the urls in our actual responses yet - if we haven't then it's fine to leave these pointing to the old urls

**Review by adadigital — APPROVED** [review]

File: specification/schemas/types/LinksError.yaml (hunk 1 above)
Comment (tdroza-nhs) [reply specification/schemas/types/LinksError.yaml:7]: Investigated by David - The API response still returns the old `communications-manager` path so the docs are correct that they match this. The urls will be updated by CCM-3068

**Pull request merged**